src/components/Thanks/BadgesCustomization.vue

```diff
@@ -252,6 +346,8 @@ export default {
 			isBlurred: true,
 			isMobileLayout: false,
 			wiggle: false,
+			badgeRevealed: false,
+			newBgMounted: false,
```

Comment:
I think a name like `badgeBlurRevealCompleted` might make more sense. Using "Mounted" makes me think it's related to Vue `mounted`, so I had to read through to figure out what this meant.